internal/pkg/githubapi/github.go

```diff
@@ -104,6 +104,30 @@ func (ghPrClientDetails *GhPrClientDetails) getBlameURLPrefix() string {
 	return fmt.Sprintf("%s/%s/%s/blame", githubHost, ghPrClientDetails.Owner, ghPrClientDetails.Repo)
 }
 
+// This function is used to check if we should display the sync branch checkbox in the PR comment.
```

Comment:
I suggest using [godoc](https://go.dev/doc/comment#func) format meaning instead of `this` use the name of the function to start.